std_msgs/Header header
geometry_msgs/PoseArray obj_poses
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status

geometry_msgs/Pose tgt_nav_pose
geometry_msgs/Pose tgt_pre_grasp_pose
geometry_msgs/Pose tgt_grasp_pose
